GitUp/Application/AppDelegate.m

```diff
@@ -560,7 +560,7 @@ - (IBAction)newRepository:(id)sender {
   if ([savePanel runModal] == NSFileHandlingPanelOKButton) {
     NSString* path = savePanel.URL.path;
     NSError* error;
-    if (![[NSFileManager defaultManager] fileExistsAtPath:path] || [[NSFileManager defaultManager] moveItemAtPathToTrash:path error:&error]) {
+    if (![[NSFileManager defaultManager] fileExistsAtPath:path followSymlink:NO] || [[NSFileManager defaultManager] moveItemAtPathToTrash:path error:&error]) {
```

Comment:
Well let's sync this through: what is clear is that both APIs must have the same behavior with regard to symlinks. The question is then: what should be the correct behavior? There are two cases:
- symlink in the last path component 
- symlink anywhere else

I think symlink in the last path component is when you need an option to follow or not. For symlinks elsewhere, they should always be followed because the alternative doesn't make much sense.